tests/Feature/Sensors/JobAttemptSensorTest.php

```diff
@@ -30,17 +31,30 @@
 
     setTraceId('0d3ca349-e222-4982-ac23-2343692de258');
     Config::set('queue.default', 'database');
+    Redis::command('FLUSHALL');
 })->skip(version_compare(Application::VERSION, '11.0.0', '<'), 'Laravel 10 support is pending');
 
-it('ingests processed job attempts', function () {
+$workCommands = [
+    'queue:work',
+    'horizon:work',
+];
```

Comment:
We now run all tests against `queue:work` and `horizon:work` to ensure we capture events consistently across both systems.

Reply:
Self-resolving to merge after approvals.